dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -197,6 +197,24 @@
 		status = "disabled";
 	};
 
+	flexcomm14: flexcomm@126000 {
+		compatible = "nxp,lpc-spi";
+		reg = <0x126000 0x1000>;
+		interrupts = <20 0>;
+		label = "FLEXCOMM_14";
+		clocks = <&clkctl1 MCUX_FLEXCOMM14_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm15: flexcomm@127000 {
```

Comment:
Flexcomm 15 is also called PMIC I2C in the reference manual, we could label this a `pmic_i2c`